pkgs/servers/matrix-synapse/sliding-sync/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "matrix-sliding-sync";
+  version = "0.99.3";
+
+  src = fetchFromGitHub {
+    owner = "matrix-org";
+    repo = "sliding-sync";
+    rev = "v${version}";
+    hash = "sha256-lmmOq0gkvrIXQmy3rbTga0cC85t0LWjDOqrH1NWUpdA=";
+  };
+
+  vendorHash = "sha256-447P2TbBUEHmHubHiiZCrFVCj2/tmEuYFzLo27UyCk4=";
+
+  postInstall = ''
+    rm $out/bin/testmigration
+  '';
+
```

```suggestion

  ldflags = [
    "-X main.GitCommit=${src.rev}"
  ];

```

Fixed version reporting.

Before:
```
❯ result/bin/syncv3
Sync v3 [0.99.3] ()
```

After:
```
❯ result/bin/syncv3
Sync v3 [0.99.3] (v0.99.3)
```

https://github.com/matrix-org/sliding-sync/blob/57c25abc802e8b08f8662dce7791f60b00228868/Dockerfile#L11
https://github.com/matrix-org/sliding-sync/blob/57c25abc802e8b08f8662dce7791f60b00228868/cmd/syncv3/main.go#L23